demo/guides/evidence-learning.html

```
@@ -0,0 +1,124 @@
+              <li>MemoryProposal self-confirmed from model or tool output.</li>
+              <li>SkillProposal activation without review.</li>
+              <li>SkillProposal tool-access expansion without Policy review.</li>
+              <li>OutcomeReport without LearningRecord reference.</li>
+              <li>OutcomeReport whose source WorkSession is not terminal.</li>
+            </ul>
+          </article>
+
+          <article class="spec-panel">
+            <h2>Learning loop</h2>
+            <pre class="code-sample"><code>WorkSession completed
+  -> EvidenceManifest exported
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)